style/index.css

```diff
@@ -38,6 +38,30 @@
   height: 100%;
 }
 
+.jp-PDFJSPageNumber {
+  display: flex;
+  align-items: center;
+  padding: 0 6px;
+}
+
+.jp-PDFJSPageNumber input {
+  border: 1px solid var(--jp-border-color0);
+  background-color: var(--jp-input-background);
```

Comment:
The text color should also be set here: in the dark theme it is a bit tough to see the number.

Reply:
Fixed.